packages/utils/src/index.ts

```diff
@@ -7,7 +7,6 @@ export * from './internals/memory-info';
 export * from './internals/debug';
 export * as social from './internals/social';
 export * from './internals/typedefs';
-export * from './internals/error_tracker';
```

Comment:
we should reexport it here, otherwise its technically breaking

Reply:
We moved it to core package, so I had to remove it.

I recreated the old `error_tracker.ts` file inside utils and exported it, [packages/utils/src/index.ts](https://github.com/apify/crawlee/pull/2332/files/e56e1cb34ef6f2e95bcdf3af15b524e385aaba5a#diff-4de4f541bddba72ba0a8692d8e366b2b8498d0a1bae6d3cb225fd3346571d567), Yes or No?

Reply:
> Yes or No?

next time dont mark the discussion as resolved if you are asking a question... 

lets not deal with this now, and hope it wont break anyone, if so, we can port it back to its initial location

Reply:
Sorry, marked it resolved by mistake